packages/master/src/routes.js

```diff
@@ -117,7 +117,7 @@ function getPlugins(req, res) {
 		if (web.main === "remoteEntry.js") {
 			web.error = "Incompatible old remoteEntry.js entrypoint.";
 		}
-		plugins.push({ name, version: pluginInfo.version, enabled, loaded, web });
+		plugins.push({ name, version: pluginInfo.version, enabled, loaded, web, requirePath: pluginInfo.requirePath });
```

Comment:
I don't like that you leak the full require path in a unauthenticated endpoint when what you actually want is is to include package.name if it equals requirePath and package.private is not true. This should also use the `snake_case` convention like everything else that goes over the network.